Dockerfile

```diff
@@ -1,18 +1,26 @@
-# Take advantage of a multi-stage build to avoid dev-dependencies in the final image
-FROM node:16-alpine AS builder
+FROM node:16-alpine AS build
+ENV NODE_ENV=development
 
 WORKDIR /app
 
 COPY . .
 
-RUN yarn
+RUN yarn install
 RUN yarn build
```

Comment:
I tried to reduce the build duration by this:

```Dockerfile
COPY ./package.json ./
RUN yarn install

COPY . .
RUN yarn build
```

([ref](https://stackoverflow.com/a/67333861))

but it did not work since this is monorepo and the submodules has many `package.json`s 🥲